app/models/solidus_stripe/payment_method.rb

```diff
@@ -164,11 +164,19 @@ def payment_profiles_supported?
     end
 
     def stripe_dashboard_url(payment)
-      # TODO: handle when the payment doesn't exist yet in Stripe, but we only have the setup intent
-      intent_id = payment.transaction_id
       path_prefix = '/test' if preferred_test_mode
 
-      "https://dashboard.stripe.com#{path_prefix}/payments/#{intent_id}"
+      intent_id =
+        payment.transaction_id ||
```

Comment:
Hey, @elia, haven't we stopped storing the intent id on the payment model? :thinking:

Reply:
No, we still provide it as the transaction id in the gateway responses, for setup intents solidus will attach it to the payment. No strong feelings, but this is the most "solidus compliant" way I think.

Reply:
Oh, right. Thanks for clarifying.